ui/src/wide_article_preview.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Model, S } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { Component, XComponents } from './form'
+import { cards } from './layout'
+import { clas, cssVar, px } from './theme'
+import { bond, wave } from './ui'
+import { XPersona } from './persona'
+
+const css = stylesheet({
+  card: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: 24,
+  },
+  clickable: {
+    cursor: 'pointer'
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-end',
+    marginBottom: 24,
+    $nest: {
+      '.ms-Persona-primaryText': {
+        fontSize: 16,
+        lineHeight: px(22),
+        fontWeight: 500,
+      },
+      '.ms-Persona-secondaryText': {
+        fontSize: 14,
+        lineHeight: px(20),
+        color: cssVar('$text7')
+      },
+      '.ms-Persona-image': {
+        height: '40px !important',
```

Comment:
nit: You can use a util func
```suggestion
        height: important('40px'),
```